src/tabulate.js

```diff
@@ -4,8 +4,9 @@ import { th, tr, td, table, tbody, a, b, span, fragment } from "./html"
 export function tabulate(lcov, options) {
 	const head = tr(
 		th("File"),
+		th("Statements"),
```

Comment:
Optional: similar to Funcs, I suggest
```suggestion
		th("Stmts"),
```

Reply:
I agree here, even if we are to change this, it should not be part of this PR.

Reply:
Changed as requested.